introduction.md

```diff
@@ -0,0 +1,4 @@
+Hello, This is my fork of the dojo_rules repo as per  instruction.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.